js/initcontext.go

```diff
@@ -214,6 +214,10 @@ func (i *InitContext) requireModule(name string) (goja.Value, error) {
 		mod = perInstance.NewModuleInstancePerVU()
 	}
```

Comment:
We probably should drop this lines and the `modules.HasModuleInstancePerVU` interface :thinking: